android/src/main/java/io/customer/reactnative/sdk/messagingpush/RNCIOPushMessaging.kt

```diff
@@ -136,9 +145,34 @@ class RNCIOPushMessaging(
             }
             true // as this permission listener can be removed now
         }
+
         else -> false // desired permission not yet granted, so we will keep the listener
     }
 
+    override fun onActivityResult(
+        activity: Activity?,
+        requestCode: Int,
+        resultCode: Int,
+        intent: Intent?,
+    ) {
+        // Nothing required here
```

Comment:
should we maybe call `super` here?

Reply:
🤔 I think it isn't needed because this actually is coming from `com.facebook.react.bridge.ActivityEventListener` interface which looks like have no default implementation. Also, from the [docs](https://reactnative.dev/docs/native-modules-android#getting-activity-result-from-startactivityforresult), doesn't look like it's required.